pkg/raft/log_unstable_test.go

```diff
@@ -110,81 +110,6 @@ func TestLastIndex(t *testing.T) {
 	}
 }
 
-func TestUnstableMaybeTerm(t *testing.T) {
```

Comment:
This is covered by `logSlice.termAt` and `raftLog.term` tests. We no longer need `unstable.maybeTerm`, since it got inlined into is only user - `raftLog.term`.